custom-checks/pmd/src/test/resources/pmd/ruleset/customrules.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="Classpath Ruleset" xmlns="http://pmd.sourceforge.net/ruleset/2.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://pmd.sourceforge.net/ruleset/2.0.0 http://pmd.sourceforge.net/ruleset_2_0_0.xsd">
+  <description>Rule set that contains custom defined rules regarding the .classpath files</description>
+  <rule class="org.openhab.tools.analysis.pmd.UseSLF4JLoggerRule" name="UseSLF4JLoggerRule"/>
+  <rule class="org.openhab.tools.analysis.pmd.WhileLoopsMustUseBracesRule" name="WhileLoopsMustUseBracesRule"/>
```

Comment:
This rule was made only for demo purposes (show the developer how a new PMD check may me implemented).

Please remove it from the ruleset.